### PR TITLE
allow overrides for helm scylla-operator replicas

### PR DESCRIPTION
**Pull request by saada**
**Description of your changes:**

Allow helm overrides for scylla operator replicas for operator and webhook server.

### CODE REVIEW

File: helm/scylla-operator/values.yaml [hunk 1]
```diff
@@ -49,6 +52,9 @@ serviceAccount:
 ## SecurityContext holds pod-level security attributes and common container settings.
 securityContext: {}
 
+# Replicas for Webhook Server
```
Comment (tnozicka) [inline helm/scylla-operator/values.yaml:55]: Although it's not surprising, please explicitly add a warning comment that setting replicas to a lower value then 2 means it won't be HA and creations or updates of Scylla CRs will fail during operator upgrades or disruptions.

File: helm/scylla-operator/values.yaml [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Operator replicas use leader election
+replicas: 2
```
Comment (tnozicka) [inline helm/scylla-operator/values.yaml:2]: Although it's not surprising, please explicitly add a warning comment that setting replicas to a lower value then 2 means it won't be HA and won't react during operator upgrades or disruptions.

**Review by tnozicka — COMMENTED** [review]
Does this also need to handle PDBs corresponding to it when the replicas are set to 1?

File: helm/scylla-operator/values.yaml (hunk 1 above)
Comment (saada) [reply helm/scylla-operator/values.yaml]: Thanks for your feedback! I added a comment and added a check before creating pdbs

File: helm/scylla-operator/values.yaml (hunk 2 above)
Comment (saada) [reply helm/scylla-operator/values.yaml:4]: Thanks for your feedback! I added a comment and added a check before creating pdbs

**Review by zimnx — APPROVED** [review]
lgtm, please rebase on current master

**Comment by saada** [comment]
> lgtm, please rebase on current master

done 👍🏽

**Review by zimnx — APPROVED** [review]

**Pull request merged**